test/advection_test/test_host.F90

```diff
@@ -462,6 +576,154 @@ subroutine test_host(retval, test_suites)
          !Reset error flag to continue testing other properties:
          errflg = 0
       end if
+      !Check moist mixing ratio for a dynamic constituent
+      call const_props(index_dyn2)%is_dry(const_log, errflg, errmsg)
+      if (errflg /= 0) then
+         write(6, '(a,i0,a,a,i0,/,a)') "ERROR: Error, ", errflg, " trying ",  &
+              "to get dry prop for dyn_const2 index = ", index_dyn2, trim(errmsg)
+         errflg_final = -1 !Notify test script that a failure occurred
+      end if
+      if (errflg == 0) then
+         if (const_log) then
+            write(6, *) "ERROR: dyn_const2 is dry and should be moist"
+            errflg_final = -1
+         end if
+      else
+         !Reset error flag to continue testing other properties:
+         errflg = 0
+      end if
+
+      !-------------------
+
+      !-------------------
```

Comment:
Have we decided on a style for comments? I personally prefer a whitespace after the `!`. The `black` formatter used for Python code in spack also mandates white spaces after `#`.

Reply:
I agree that the white space after the "!" looks best. Updated comments in this file